### ISSUE TITLE
ImportError: cannot import name 'Huber' from 'sklearn.linear_model._sgd_fast' 

### ISSUE THREAD

**Issue by shsunmoonlee** [segment-any-text/wtpsplit#144]
May I ask what's the issue? 
i'm using python 3.12. mac m2 pro. 
uv init
uv sync

// pyproject.toml
```toml
[project]
name = "llm-pipeline"
version = "0.1.0"
description = "Add your description here"
readme = "README.md"
requires-python = ">=3.12"
dependencies = [
    "scikit-learn>=1.6.0",
    "torch>=2.5.1",
    "torchaudio>=2.5.1",
    "torchvision>=0.20.1",
    "transformers>=4.45.2",
    "wtpsplit>=2.1.1",
]
```

hello.py
```python
from wtpsplit import SaT

def main():
    # print("Hello from goodlisten-llm-pipeline!")

    sat = SaT("sat-3l")
    # optionally run on GPU for better performance
    # also supports TPUs via e.g. sat.to("xla:0"), in that case pass `pad_last_batch=True` to sat.split
    sat.half().to("cuda")

    sat.split("This is a test This is another test.")
    # returns ["This is a test ", "This is another test."]

    # do this instead of calling sat.split on every text individually for much better performance
    sat.split(["This is a test This is another test.", "And some more texts..."])
    # returns an iterator yielding lists of sentences for every text

    # use our '-sm' models for general sentence segmentation tasks
    sat_sm = SaT("sat-3l-sm")
    sat_sm.half().to("cuda") # optional, see above
    sat_sm.split("this is a test this is another test")
    # returns ["this is a test ", "this is another test"]

    # use trained lora modules for strong adaptation to language & domain/style
    sat_adapted = SaT("sat-3l", style_or_domain="ud", language="en")
    sat_adapted.half().to("cuda") # optional, see above
    sat_adapted.split("This is a test This is another test.")
    # returns ['This is a test ', 'This is another test']


if __name__ == "__main__":
    main()

```

```
Traceback (most recent call last):
  File "/Users/seunghun.lee/Code/test-python/hello.py", line 1, in <module>
    from wtpsplit import SaT
  File "/Users/seunghun.lee/Code/test-python/.venv/lib/python3.13/site-packages/wtpsplit/__init__.py", line 12, in <module>
    import skops.io as sio
  File "/Users/seunghun.lee/Code/test-python/.venv/lib/python3.13/site-packages/skops/io/__init__.py", line 1, in <module>
    from ._persist import dump, dumps, get_untrusted_types, load, loads
  File "/Users/seunghun.lee/Code/test-python/.venv/lib/python3.13/site-packages/skops/io/_persist.py", line 22, in <module>
    module = importlib.import_module(module_name, package="skops.io")
  File "/opt/homebrew/Cellar/python@3.13/3.13.1/Frameworks/Python.framework/Versions/3.13/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/seunghun.lee/Code/test-python/.venv/lib/python3.13/site-packages/skops/io/_sklearn.py", line 15, in <module>
    from sklearn.linear_model._sgd_fast import (
    ...<9 lines>...
    )
ImportError: cannot import name 'Huber' from 'sklearn.linear_model._sgd_fast' (/Users/seunghun.lee/Code/test-python/.venv/lib/python3.13/site-packages/sklearn/linear_model/_sgd_fast.cpython-313-darwin.so)
```

**Comment by markus583** [comment]
Hi, which sklearn version are you using?

**Comment by shsunmoonlee** [comment]
Hi! 

i've edited my initial post above.
I simply created new project 
using python 3.12. mac m2 pro.
uv init
uv add wtpsplit 
and others

**Comment by markus583** [comment]
Alright, thanks for the additional info!
This is an issue of the skops dependency (see this [nltk issue](https://github.com/nltk/nltk/issues/3346) and this [skops ](https://github.com/skops-dev/skops/issues/454)issue). It has already been fixed with version 0.11. Please update the dependency:
`pip install skops --upgrade`

I just checked myself and I can reproduce this error with skops==0.10. If I upgrade to skops==0.11, the import works perfectly fine again.
